api/document/tests/serializers/doc_cursor_test.py

```diff
@@ -275,3 +287,18 @@ def test_content_outside():
     assert [c['content_type'] for c in content] == [
         'external_link', '__text__', 'external_link',
     ]
+
+
+def test_children_field_to_internal_value_works():
+    # TODO: We really shouldn't *have* to supply a type_emblem here,
+    # because the deserializer will ultimately pass the payload through
+    # DocCursor, which will auto-assign one.
```

Comment:
I think we can replace the auto-generated `type_emblem` field of `DocCursorSerializer` with one that allows blank.

Reply:
Cool... this is making me start to wonder if it's better to just make `DocCursorSerializer` a standard `Serializer`, rather than turn it into a `DocNode` serializer as I'd suggested in an earlier PR.  It seems a number of the validations performed by the `DocNode` model aren't actually going to be needed by the deserializer due to all the stuff it's auto-populating.